presto-parser/src/main/java/com/facebook/presto/sql/tree/RoutineBody.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+public class RoutineBody
+        extends Statement
+{
+    private final Optional<Expression> returnExpression;
+    private final Optional<ExternalBodyReference> externalBodyReference;
```

Comment:
An additional comment regarding the simplification, I said it is temporary because by doing
```Java
public class Return extends RoutineBody {}
```
`Return` is no longer a `Statement`, but only a `Node`. This is ok for now as we're not using the `Statement` aspect of the return statement.

Reply:
Gonna take the simplified version. Don't see much benefit for adding so many classes when we are not planning to add those functionalities any time soon. 😛